src/lightning_lite/utilities/device_parser.py

```diff
@@ -4,8 +4,8 @@
 
 import torch
 
-# TODO(lite): Fix the imports
```

Comment:
Keep the TODO for the line below?

Reply:
added back, but already resolved in #14555